hardware/bluetooth/control/bluetooth_control_command.ksy

```diff
@@ -0,0 +1,42 @@
+meta:
+  id: bluetooth_control_command
+  title: Bluetooth Host Controller Interface Control Protocol
+  license: Unlicense
+  file-extension:
+    - hcd
```

Comment:
See https://github.com/kaitai-io/kaitai_struct_formats/pull/305#discussion_r456796225:

```suggestion
  file-extension: hcd
```

Reply:
Isn't `bluetooth_control_command` meant to be an import-only spec? Is there some version of a binary file with `.hcd` extension that can be directly parsed using the `bluetooth_control_command`, without using the `bluetooth_hcd` format?

If the `bluetooth_control_command` is not meant to represent the full `.hcd` format, then there IMHO shouldn't be the `file-extension` key at all.

Reply:
In fact originally there was only one file with a mess inside, then I have copied it into multiple places and removed the irrelevant parts and made other changes. It seems I have forgotten to remove the `file-extension` from it.